### PR TITLE
Feature: Stream OR-Operator UI requirements

### PR DESCRIPTION
**Pull request by dennisoelkers**
This PR adds the ability to update the matching type for existing streams.

### CODE REVIEW

File: graylog2-rest-client/src/main/java/org/graylog2/restclient/models/Stream.java [hunk 1]
```diff
@@ -39,8 +39,9 @@
 import java.util.List;
 import java.util.Map;
 
-public class Stream {
+import static com.google.common.base.Strings.emptyToNull;
```
Comment (edmundoa) [inline graylog2-rest-client/src/main/java/org/graylog2/restclient/models/Stream.java]: I couldn't find any usages of `emptyToNull` on this class, can we get rid of this import?

**Comment by edmundoa** [comment]
Other than the unused `import`, the changes look good to me.

**Comment by dennisoelkers** [comment]
Thanks for reviewing! :+1:

**Comment by edmundoa** [comment]
Looking good :+1:

**Pull request merged**